src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,42 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
```

Comment:
I think you can add error text from Swagger here - it is better formulated there